components/prism-sas.js

```diff
@@ -10,25 +23,152 @@ Prism.languages.sas = {
 			'punctuation': /;/
 		}
 	},
+	'procSQL': {
+		pattern: /(^|\n)(?:proc\s+sql(?:\s+[\w|=]+)?);([\s\S]+(?:\r?\n|\r)(?=((?:proc\s\w+|quit|run|data(?!\=));))((?:proc\s\w+|quit|run|data(?!\=));)|(?:\r?\n|\r)[\s\S]+;)/im,
+		inside: {
+			'fullStep': {
+				pattern: /(^|\n)(?:proc\s+sql(?:\s+[\w|=]+)?);/i,
+				inside: {
+					'step': {
+						pattern: /(?:proc\s\w+|quit|run|data(?!\=))\b/i,
```

Comment:
Aren't __all__ `proc\s\w+` supposed to be `proc\s+\w+`?

Reply:
Yep. Will fix.